qt_configure.bzl

```diff
@@ -17,6 +17,13 @@ def qt_autoconf_impl(repository_ctx):
     if os_name.find("windows") != -1:
         # Inside this folder, in Windows you can find include, lib and bin folder
         default_qt_path = "C:\\\\Qt\\\\5.9.9\\\\msvc2017_64\\\\"
+        # If predefined path does not exist search for an alternative e.g. "C:\\\\Qt\\\\5.12.10\\\\msvc2017_64\\\\"
+        if not repository_ctx.path(default_qt_path).exists:
+            win_path_env = _get_env_var(repository_ctx, "PATH")
+            start_index = win_path_env.index("C:\\Qt\\5.")
```

Comment:
As-is, the call to `string.index()` will [raise an error](https://docs.bazel.build/versions/3.5.0/skylark/lib/string.html#index) if there's no qt found in $PATH. This is good, but it may be better to catch that error and print a more useful message like "No Qt 5 found in $PATH".